### ISSUE TITLE
Adding scheduling rule for service name is not intuitive

### ISSUE THREAD

**Issue by deniseschannon** [rancherio/rancher#1286]
v0.24.0

Steps I did:
1. Create service A
2. Create service B with rule of "host have a service with name A". 

Problem I encountered: I did not know that I had to put "project name/service name" as the value. 

1. We should include some UI tip that indicates if service name is selected they need to put in the "project name/service name" as the value. Optional: a tip that the list of existing services are available in the drop down?
2. There is a dropdown for existing services, but only AFTER they have been started. We should have all services in the dropdown for whenever they are created. Since I might want to set up everything before I click on launch services, none of my services will show up in the drop down.

**Comment by vincent99** [comment]
You have to start them because all the dropdowns are based on the list of existing labels and values on existing containers.. To do it ahead of time I'd have to guess what the values are going to be, which makes no sense for some fields and could be done for others.

**Comment by deniseschannon** [comment]
fair enough, can we make a tip that the service name must be in the format of project name/service name (if someone adds for a service not created yet?)

**Comment by deniseschannon** [comment]
pre-duplicate of #1382
